### ISSUE TITLE
"Get Vlocity Dependencies" error handling when no Vlocity selected

### ISSUE THREAD

**Issue by microweii** [swongcopado/dxcau2vlocity#18]
When only SFDC (and _not_ Vlocity) items are selected prior to executing "Get Vlocity Dependencies" job, one of the following behaviours is observed:

- Stuck at Initializing
- Stuck at Starting
- Unexpected error occurs
![image](https://user-images.githubusercontent.com/71686297/183345490-80dcf0ac-00c5-40e4-a374-65714f1329a4.png)

When the "unexpected error" is encountered, the session must be refreshed. Otherwise rerunning the "Get Vlocity Dependencies" job shows the same modal.

**Comment by swongcopado** [comment]
Thanks. May I know what's the purpose of this test?
The Get Vlocity Dependencies button is to pull in Vlocity Dependencies, not
SF metadata.
If you don't select any Vlocity components and try to run the Get Vlocity
Dependencies, I would expect Copado to throw an error.

**Comment by microweii** [comment]
Agreed, error is expected. The minor inconvenience is that when the error is thrown, the user would have to refresh the page/session. Otherwise, selecting the Vlocity components and re-executing the get dependencies function just spits the same error modal from before, without actually re-running the job.

This wasn't specifically tested for, but noticed on accident due to confusion between LWC component and actual OmniScript/FlexCard names.

Not a priority, but noting down for future reference.

**Comment by swongcopado** [comment]
OK. If it's ok, let's close this issue.
